cypress/e2e/admin/administrateUsers.feature

```diff
@@ -1,6 +1,7 @@
 @regression_test
 @stable_test
-@schedule_run
```

Comment:
why removed `@schedule_run` tag in all feature files? I don't think we have to remove them from scheduled workflow.